src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterfaceProgress.cs

```diff
@@ -92,6 +99,21 @@ class ConsoleHostUserInterface : System.Management.Automation.Host.PSHostUserInt
             {
                 // Update the progress pane only when the timer set up the update flag or WriteProgress is completed.
                 // As a result, we do not block WriteProgress and whole script and eliminate unnecessary console locks and updates.
+                if (ExperimentalFeature.IsEnabled(ExperimentalFeature.PSAnsiProgressFeatureName) && PSStyle.Instance.Progress.UseOSCIndicator)
+                {
+                    int percentComplete = record.PercentComplete;
+                    if (percentComplete < 0)
+                    {
+                        // Write-Progress allows for negative percent complete, but not greater than 100
+                        // but OSC sequence is limited from 0 to 100.
+                        percentComplete = 0;
+                    }
+
+                    // OSC sequence to turn on progress indicator
+                    // https://github.com/microsoft/terminal/issues/6700
```

Comment:
Is there an official document for the OSC sequences supported in Windows Terminal?

Reply:
None I could find other than the github issue